src/course-unit/add-component/messages.js

```diff
@@ -13,6 +13,18 @@ const messages = defineMessages({
     id: 'course-authoring.course-unit.modal.button.text',
     defaultMessage: 'Select',
   },
+  singleComponentPickerModalTitle: {
+    id: 'course-authoring.course-unit.modal.single-title.text',
+    defaultMessage: 'Select component',
```

Comment:
@navinkarkera Could you add the `description` to these messages? It is good practice to put the description

Reply:
@ChrisChV Ahh, yes. Did not notice the missing description field. Thanks, added it here: https://github.com/openedx/frontend-app-authoring/pull/1522/commits/a63f1768362e57df7f574577bd3ba4179f1bdf71